account_mass_reconcile_by_purchase_line/models/mass_reconcile.py

```diff
@@ -0,0 +1,17 @@
+# © 2015-18 Eficent Business and IT Consulting Services S.L. (www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import models
+
+
+class AccountMassReconcileMethod(models.Model):
+    _inherit = 'account.mass.reconcile.method'
+
+    @staticmethod
+    def _get_reconcilation_methods():
+        methods = super()._get_reconcilation_methods()
```

Comment:
File "/home/odoo/build/OCA/bank-statement-reconcile/account_mass_reconcile/models/mass_reconcile.py", line 85, in _selection_name
    return self._get_reconcilation_methods()
  File "/home/odoo/build/OCA/bank-statement-reconcile/account_mass_reconcile_by_purchase_line/models/mass_reconcile.py", line 12, in _get_reconcilation_methods
    methods = super()._get_reconcilation_methods()
RuntimeError: super(): no arguments

Reply:
Fixed. Could you try again, @aheficent ?

Reply:
what's wrong with runbot?